sdk/go/common/apitype/updates.go

```diff
@@ -216,6 +216,17 @@ type PatchUpdateCheckpointRequest struct {
 	Deployment json.RawMessage `json:"deployment,omitempty"`
 }
 
+// PatchUpdateUntypedCheckpointRequest defines the body of a request to a patch update checkpoint endpoint of the service
+// API, where `UntypedDeloyment` is an `apitype.UntypedDeployment`. Whereas the PatchUpdateCheckpointRequest API is subject
+// to the service reformatting how a checkpoint is persisted, PatchUpdateUntypedCheckpointRequest enables the CLI to define
+// the exact format.
+// Designed to be compatible with the PatchUpdateCheckpointDeltaRequest API, where formatting is critical in calculating
+// textual diffs.
+type PatchUpdateUntypedCheckpointRequest struct {
```

Comment:
Naming is hard, very open to suggestions. `PatchUpdateCheckpointRequestV2`?

Reply:
Bike shedding, of course, but I would throw in `Verbatim` as the descriptor here. As in this endpoint stores a checkpoint _verbatim_, without modification.

`PatchUpdateVerbatimCheckpointRequest`?

Reply:
thats a good word